dashboard/app/models/course.rb

```diff
@@ -77,7 +77,8 @@ def persist_strings_and_scripts_changes(scripts, course_strings)
   end
 
   def write_serialization
-    return unless Rails.application.config.levelbuilder_mode
+    # Only save non-plc course, and only in LB mode
```

Comment:
Because they are still a fairly different thing. The serialization of a regular course is primarily a set of scripts. PLC courses instead get created when a script specifies something like `professional_learning_course 'All The PLC Thing'`